docs/src/pages/[platform]/components/shared/GetFieldControls.tsx

```diff
@@ -32,11 +33,12 @@ export const GetFieldControls = ({ fields }: GetFieldControlsProps) => {
           />
         ) : (
           <TextField
+            key={`${name}-${idx}`}
             name={name}
             placeholder={`Set ${name}`}
             value={value}
             label={name}
-            onChange={(event: any) => {
+            onChange={(event: React.BaseSyntheticEvent) => {
```

Comment:
[BaseSyntheticEvent](https://reactjs.org/docs/events.html) includes all the basic events.

`ChangeEventHandler<HTMLInputElement>` doesn't work here because it's an `Event` not an `EventHandler`

Reply:
Oh that's my bad hehe